decoder/src/lib.rs

```diff
@@ -113,8 +133,9 @@ impl Table {
         self.entries.is_empty()
     }
 
+    /// Iterates over the raw symbols of the table entries
     pub fn symbols<'s>(&'s self) -> impl Iterator<Item = &'s str> + 's {
```

Comment:
this method relies on `Item` being a 'raw symbol' (as it appears in the `.symtab`)

Reply:
We could rename it to `raw_symbols` if that doesn't cause too much churn, to make it a bit clearer

Reply:
sure!